govuk-frontend.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=govuk-frontend.js.map
```

Comment:
As mentioned, can we import the usage through npm? In addition, the source map for this file (as defined) will not exist without node import at build time and the file will not be kept up to date with version changes.

Reply:
Yes, agreed. I might draft another PR for that and do a bit of a whip around to see what the others are using in their blocks, if they require all of this or if I just pull out the accordion JS.

Reply:
I'm reluctant to push the file into the primary branch (aka master!) Can you use this:
`import { Accordion } from 'govuk-frontend';` That will get the JS in to your component.

Reply:
And then instantiate the JS - maybe by doing this: 
`new Accordion(document.querySelector('[data-module="govuk-accordion"]')).init();` needs implementing but along those lines.

Reply:
https://wordpress.stackexchange.com/questions/328536/load-css-javascript-in-frontend-conditionally-if-block-is-used

Reply:
We could have another JS file in the component that imports the required code (Accordion in your case) and then compiles it into the build directory as `mojblocks.js`...  maybe?
@RobLoweMOJ @BNewing : will you guys be using custom JS code that is needed on the front end too? Should we implement something lick this?

Reply:
`frontend.js`:

```
import { Accordion } from 'govuk-frontend';
new Accordion(document.querySelector('[data-module="govuk-accordion"]')).init();
```
Compile the JS using mix...
```
mix_.js('src/**/frontend.js');
```

Reply:
Excellent, yes that looks good to me

Reply:
The updates look great mate well done! Can you just remove this file?